.github/workflows/check_style.yml

```diff
@@ -16,9 +16,8 @@ jobs:
       - name: Checkout astyle
         uses: actions/checkout@v4
         with:
-          repository: szepeviktor/astyle
+          repository: lvgl/astyle
           path: astyle
-          ref: v3.4.12
```

Comment:
Should we use specific version instead. The developers can easily check version that CI uses from here. Using `master` is not recommended.